base/templates/base/public_base.html

```diff
@@ -429,20 +429,46 @@ <h3>{{self.friendly_name}}Social Media</h3>
                                 {% endif %}
                             </ul>
                         </div>
-                    {% endif %}
-
-                    {% if libra %}
+			{% endif %}
+			
+			{% if libra %}
                         <div class="rightside-mod col-xs-12 col-sm-6 col-md-12">
                             <h3>Print</h3>
                             <div class="libra-link-wrap">
                                 <p><a href="{% pageurl libra %}">{{libra}}</a></p>
                             </div>
                         </div>
-                    {% endif %}
+			{% endif %}
 
-                    {% if contacts %}
+			{% if contacts %}
                         {% list_contacts_in_sidebar contacts 'Media Contact' %}
-                    {% endif %}
+			{% endif %}
+
+			{% if self.category %}
+			<h3>
+			    Subscribe to {{ self.category }} <br>
+			</h3>
+			<div class="rightside-mod col-xs-12 col-sm-6 col-md-12">
+			    <a href="/rss/{{ self.slug }}/">
+				<i class="fa fa-rss-square" aria-hidden="true"></i>
+				RSS feed
+			    </a>
+			</div>
+			{% elif self.news_feed_api %}
```

Comment:
The recommended Django best practice is to indent html in dynamic code blocks, even if it will ultimately render "over-indented" html in the browser. So, that means we would normally want to indent the html within the if clause. I can see we haven't done that consistently throughout this template (my bad) but we probably should.

Reply:
Ok, these tab/whitespace issues should be fixed in the latest commit.  Meanwhile, I'll work on setting my Emacs up to automatically indent Django templates according to our indentation spec.